README.md

```diff
@@ -358,12 +357,13 @@ NamedColor   : Translucent
 The generated code will look like this:
 
 ```swift
-extension UIColor {
+// The Color type below is typealias'ed to UIColor on iOS and NSColor on OSX
+extension Color {
   /* Private Implementation details */
   ...
 }
 
-enum ColorName : UInt32 {
+enum ColorName {
```

Comment:
That `: UInt32` is generated when we use the old template now renamed `colors-rawValue.stencil`, which generated  `case Translucent = 0xffffffcc` cases.

Now that the new default template `colors-default.stencil` doesn't use `rawValues` but instead use a dictionary to map each `case` to the `UInt32` output reba value, then indeed that `: UInt32` is not part of the generated code anymore… but those `case Translucent = 0xffffffcc` should disappear as well!

So we should also update the body of that `enum` currently pasted in that README too so that the example code pasted here is still consistent with what is generated by the default template.